config/environments/development.rb.SAMPLE

```diff
@@ -39,7 +39,7 @@ Foodsoft::Application.configure do
   # Mailcatcher config, start mailcatcher from console with 'mailcatcher'
   # Mailcatcher can be installed by gem install mailcatcher
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = { address: "localhost", port: 1025 }
+  config.action_mailer.smtp_settings = { address: "mailcatcher", port: 1025 }
```

Comment:
Yep, thats a good idea. We also could do this in application.rb for all
environments at once, so we don't have to maintain multiple action
mailer configurations.